youtube_dl/extractor/keezmovies.py

```diff
@@ -122,7 +132,13 @@ def extract_format(format_url, height=None):
         }
 
     def _real_extract(self, url):
-        webpage, info = self._extract_info(url)
+        video_id, display_id = self._get_ids(url)
+        webpage = self._download_webpage_age_verified(url, display_id)
+        embed_url = self._search_regex(
+            r'<iframe\s+id="embedPlayer"\s+src="//(.+?)"', webpage, 'embed url', default=None)
+        if embed_url is not None:
+            return self.url_result(embed_url, 'Generic')
+        webpage, info = self._extract_info(url, require_http_prefix=False)
```

Comment:
Do not change the order of extraction.

Reply:
@dstftw - OK!  As we are keeping the same order of extraction, I believe we will also need to check if the `formats` collection contains any entries prior to calling `_sort_formats` - to accommodate those pages that contain only embedded urls intended for the generic extractor.  Is this reasoning correct?  Much appreciated!

Reply:
@dstftw - Where should the `fatal` flag be inserted?  I don't believe the `_sort_formats` call accepts this flag.  Thanks!

Reply:
@dstftw - Can you please clarify your comment?  Should a new `fatal` parameter also be passed to the `_sort_formats` call?  I suppose we can update the signature as `def _sort_formats(self, formats, field_preference=None, fatal=True)` to preserve the default behaviour in the other extractors.  Do you agree?

Reply:
`fatal` must be added to `_extract_info`. No changes to core code.

Reply:
@dstftw - Thanks for the clarification!